src/Components/DisplayAllArticles.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import axios from "axios";
+import ArticleTemplate from "./ArticleTemplate";
+
+class DisplayAllArticles extends Component {
+  constructor() {
+    super();
+    this.state = {
+      allArticles: []
```

Comment:
Will this property always hold "all" articles? Why not go with a more generic name `articles`?

Reply:
This is a good question.  My original thought when first planning this component was that, later on when we separate the articles into categories and then show them on the view in separate groups, that there would be a separate GET methods for each category of articles.  Therefore, pulling AllArticles would be one of those GET requests that just gets all of them.  The sorting of the articles into categories for the view, however, can probably be done more efficiently on the Client side with a single GET request to the API.  I'll change this to "articles" for now.